samples/CSharp/SourceGenerators/SourceGeneratorSamples/CSVGenerator.cs

```diff
@@ -0,0 +1,192 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using static System.Console;
+using System.Text;
+using System.Diagnostics;
+using NotVisualBasic.FileIO;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System.Collections.Immutable;
+
+#nullable enable
+
+// CsvTextFileParser from https://github.com/22222/CsvTextFieldParser adding suppression rules for default VS config
+
+#pragma warning disable IDE0008 // Use explicit type
```

Comment:
📝 This suppression should be removed and the diagnostics fixed before this is merged

Reply:
Honestly, I wondered why this sample is set up so that you are forced to give explicit types to all variables. This is not the default for VS, not what we recommend and rather cumbersome to write code this way. If it were me, I would change this sample so that one can use 'var' freely. But I didn't want to do it myself as didn't know the reasons for it.

Reply:
> Honestly, I wondered why this sample is set up so that you are forced to give explicit types to all variables.

I'm not sure _why_ it's configured this way, but changing the code style where it's already configured in a strict manner would be a separate pull request.

Reply:
I'm working with @chsienki to clean up the details on this pull request so it can be merged as new examples. We should have more information over the next week.

Reply:
Ok. I was just commenting on how strange the setup was for this sample and wondered if we should fix it at the same time. I don't think anyone would mind us doing it in one PR, but you guys decide.